roles/chameleon_mariadb/templates/mariadb.j2

```diff
@@ -9,13 +9,17 @@
 # FLUSH PRIVILEGES;
 #
 
+ROTATE="{{ mariadb_backup_file_age }}"
+
+set -o pipefail
+
+{% if not enable_mariabackup %}
```

Comment:
Do we need to keep this still? What if we removed the old way instead of preserving the two different ways of doing this?

Reply:
I think TACC has to use the old way since its MariaDB is managed externally (i.e. not deployed via kolla), right?

Reply:
Grr, you're right, I thought they were on the containerized DB already.